src/components/HistoricalContainer/HistoricalContainer.js

```diff
@@ -1,22 +1,62 @@
-import './HistoricalContainer.css';
+import React, { useState } from 'react';
 import HistoricalCards from '../HistoricalCards/HistoricalCards';
+import './HistoricalContainer.css';
 
 function HistoricalContainer({ theHistory, launchImages }) {
-  const allHistory = theHistory.filter(story => {
-    return story.links.article && !story.links.article.includes('www.spacex.com')      
-  }).map((story, i) => (
-    <HistoricalCards 
-      key={story.id}
-      title={story.title}
-      details={story.details}
-      link={story.links.article} 
-      image={launchImages[i]} 
-    /> 
-  ))
+  const [favoritesList, setFavoritesList] = useState([])
+  const [isFavorited, setIsFavorited] = useState(false)
+
+  const updateFavorites = (story) => {
+    let foundFavorite = favoritesList.find(favorite => favorite.id === story.id)
+
+    if (foundFavorite) {
+      removeFromFavorites(foundFavorite)
+    } else {
+      addToFavorites(story)
+    }
+  }
   
+  const addToFavorites = (story) => {
+    let newFavorite = {
+      id: Date.now(), ...story
+    }
+    setFavoritesList([...favoritesList, newFavorite])
+  }
+
+  const removeFromFavorites = (story) => {
+    let itemsToKeep = favoritesList.filter(favorite => favorite.id !== story.id)
+    setFavoritesList(itemsToKeep)
+  }
+
+  const toggleDisplay = () => {
```

Comment:
I don't know if `toggleDisplay` is a great name. Maybe `toggleFavorite` would be more clear?